cmd/filebrowser/main.go

```diff
@@ -63,6 +64,8 @@ func init() {
 	flag.BoolVar(&allowCommands, "allow-commands", true, "Default allow commands option for new users")
 	flag.BoolVar(&allowEdit, "allow-edit", true, "Default allow edit option for new users")
 	flag.BoolVar(&allowPublish, "allow-publish", true, "Default allow publish option for new users")
+	flag.StringVar(&auth.method, "auth.method", "default", "Switch between 'default' and 'proxy' authentication.")
+	flag.StringVar(&auth.loginHeader, "auth.login-header", "X-Forwarded-User", "The header name used for proxy authentication.")
```

Comment:
I don't know how I feel about the dot in the flag honestly. Probably replace it by `-` for consistency. @maweck why did you choose to use `.`?

Reply:
Because that is how it is done in most of the tools I work with (Maven, SonarQube, Jenkins, ..... ). In addition it allows you to "group" specific configs. Here is a sample from SonarQube:

>sonar.web.sso.enable=false
>sonar.web.sso.loginHeader=X-Forwarded-Login
>sonar.web.sso.nameHeader=X-Forwarded-Name
>sonar.web.sso.emailHeader=X-Forwarded-Email
>...
>sonar.security.realm=LDAP
>ldap.url=ldap://localhost:10389
>ldap.bindDn=cn=sonar,ou=users,o=mycompany

But sure, I can change that if you want :)

Reply:
Looking at that tools they also seem to use camelCase. It would not be a bad idea. There are some of the other options which should be also grouped.

Could you add them with dots and camelCase please? I'll make another PR to update the other ones because it actually makes sense.

Reply:
Could you also rework it in a way to include noAuth on auth object? Maybe could be renamed auth.disable

Reply:
Ref.: #497